[Task] Update support docs with submodule mkdocs SA 5307

RSAK56 opened the pull request:
# Description
1. Updated support docs with mkdocs theme common submodule
2. Shifted vocabulary to vale and removed vocabulary submodule
Fixes # (issue)
UI/UX
# Type of change
Enhancement
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] UI/UX modification
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
# Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

karl-johan-grahn reviewed (changes requested):
the logos you added are generic, why are they not part of the common theme?

RSAK56 commented:
> the logos you added are generic, why are they not part of the common theme?

They will be updated for each theme separately, when provided by marketing team or designers. But we need to configure that now, rather than later. 
So when we get those updated logos, we'll just need to replace the logo's with the updated ones rather than configuring the common theme for all docs again.

RSAK56 commented:
> The common theme should contain the default logos. For this doc set, the default logos should be used. So they should come from the common theme, and not have to be stored here.

Updated footer logos in the theme common and removed from override
